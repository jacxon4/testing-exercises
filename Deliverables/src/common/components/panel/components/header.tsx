import * as React from 'react';
const styles = require('./header.scss');

interface Props {
  title: string;
}

// Test to use identity-object-proxy to support node modules
export const Header = (props: Props) => (
  <div className={`card-header ${styles.header}`}>
    <h3 className="panel-title">{props.title}</h3>
  </div>
);
