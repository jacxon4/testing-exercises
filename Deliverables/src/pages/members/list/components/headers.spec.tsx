import * as React from 'react';
import { shallow } from 'enzyme';
import { Header } from './header';

// Useless test
describe('pages/members/list/components/header', () => {
  it('should be static', () => {
    const component = shallow(<Header />);

    expect(component).toMatchSnapshot();
  });
});
