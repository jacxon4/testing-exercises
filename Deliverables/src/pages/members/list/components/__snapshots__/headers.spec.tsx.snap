// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/header should be static 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;
